File: Challenges/Coding World Cup/2023/Day 30 - AFG vs NED - Rashid's Plan.java
public class Solution {
    static int countEvenParityDigits(String a) {
        int count = 0;
        for (int i = 0; i < a.length(); i++) {
            if ((a.charAt(i) - '0') % 2 == 0) {
                count++;
            }
        }
        return count;
    }
}

File: Challenges/Coding World Cup/2023/Day 30 - AFG vs NED - Rashid's Plan.cpp
int countEvenParityDigits(string a)
{
    // Write your code here.
    int count = 0;

    for (int i = 0; i < a.size(); i++)
    {
        int digit = a[i] - '0';
        if (digit % 2 == 0)
            count++;
    }

    return count;
}
